.github/workflows/windows-meson-msys2.yml

```diff
@@ -0,0 +1,136 @@
+name: MSYS2 builds
+
+on: [push, pull_request]
+
+jobs:
+  build_win_meson_msys2_release:
+    name: ${{ matrix.conf.name }}
+    runs-on: windows-latest
+    if: github.event_name != 'pull_request' || contains('dreamer,kcgen,ant-222,Wengier,shermp', github.actor) == false
+    
+    strategy:
+      matrix:
+        conf:
+          - name: Meson MinGW 32-bit
+            arch: i686
+            sys: MINGW32
+          - name: Meson MinGW 64-bit
```

Comment:
Suggest naming the builds by the compiler `GCC MinGW 32-bit` (or Clang, if/when).
Following a similar pattern as the others.
![2021-11-19_18-40](https://user-images.githubusercontent.com/1557255/142711743-8fdc8fdb-797c-4d63-a0bc-42f5d47a88f5.png)